apps/console/src/features/users/components/user-change-password.tsx

```diff
@@ -318,89 +341,221 @@ export const ChangePasswordComponent: FunctionComponent<ChangePasswordPropsInter
         });
     };
 
+    /**
+     * Callback function to validate password.
+     *
+     * @param valid - validation status.
+     * @param validationStatus - detailed validation status.
+     */
+    const onPasswordValidate = (valid: boolean): void => {
```

Comment:
```suggestion
    const onPasswordValidate = (isValid: boolean): void => {
```
Better variable suggestion for boolean

Reply:
[c9aa0aa](https://github.com/wso2/identity-apps/pull/5302/commits/c9aa0aa16b54a49c6820e85b03a0866f5089d38c)